### PR TITLE
feat: Add `twitch-eventsub-to-discord-webhook` community example

### PR DESCRIPTION
**Pull request by dylmye**
Hi, I have added my template as a community example :) I hope it's good enough quality, this is my first template! I appreciate any feedback as I found the docs hard to follow. I have got it working on my own serverless account.

### CODE REVIEW

**Review by pgrzesik — APPROVED** [review]
Thanks @dylmye for sharing your example 🙇

**Comment by dylmye** [comment]
@pgrzesik Are we good to revert this PR? I realise it was meant to go into v3
